packages/apt/test.sh

```diff
@@ -163,12 +163,20 @@ echo "::group::Upgrade"
 apt purge -V -y ${pgroonga_package}
 
 if apt show ${pgroonga_package} > /dev/null 2>&1; then
-  can_upgrade=yes
+  is_first_release=yes
 else
-  can_upgrade=no
+  is_first_release=no
 fi
 
-if [ "${can_upgrade}" = "yes" ]; then
+pgroonga_latest_released_version_full=$(apt info ${pgroonga_package} | \
+                                          grep Version | \
+                                          sed -E 's/^Version: ([0-9]\.[0-9]\.[0-9])-([0-9])/\1 \2/')
+pgroonga_latest_released_version=$(echo ${pgroonga_latest_released_version_full} | \
+                                     awk '{print $1}')
+pgroonga_release_number=$(echo ${pgroonga_latest_released_version_full} | \
+                            awk '{print $2}')
+
+if [ "${is_first_release}" = "yes" ]; then
```

Comment:
初回リリースのときは、対象のパッケージがないのでスキップするになるはず。
```suggestion
if [ "${is_first_release}" = "no" ]; then
```

Reply:
なおしました。

Reply:
すでに対応済みでした。すみませんmm

Reply:
いえ、ありがとうございます。たすかります。